pkg/asset/cluster/tfvars.go

```diff
@@ -1,27 +1,47 @@
 package cluster
 
 import (
+	"fmt"
 	"os"
 
+	libvirtprovider "github.com/openshift/cluster-api-provider-libvirt/pkg/apis/libvirtproviderconfig/v1alpha1"
 	"github.com/openshift/installer/pkg/asset"
 	"github.com/openshift/installer/pkg/asset/ignition/bootstrap"
 	"github.com/openshift/installer/pkg/asset/ignition/machine"
 	"github.com/openshift/installer/pkg/asset/installconfig"
+	"github.com/openshift/installer/pkg/asset/machines"
 	"github.com/openshift/installer/pkg/asset/rhcos"
 	"github.com/openshift/installer/pkg/tfvars"
+	awstfvars "github.com/openshift/installer/pkg/tfvars/aws"
+	libvirttfvars "github.com/openshift/installer/pkg/tfvars/libvirt"
+	openstacktfvars "github.com/openshift/installer/pkg/tfvars/openstack"
+	"github.com/openshift/installer/pkg/types/aws"
+	"github.com/openshift/installer/pkg/types/libvirt"
+	"github.com/openshift/installer/pkg/types/none"
+	"github.com/openshift/installer/pkg/types/openstack"
 	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	awsprovider "sigs.k8s.io/cluster-api-provider-aws/pkg/apis/awsproviderconfig/v1alpha1"
+	openstackprovider "sigs.k8s.io/cluster-api-provider-openstack/pkg/apis/openstackproviderconfig/v1alpha1"
 )
 
 const (
 	// TfVarsFileName is the filename for Terraform variables.
-	TfVarsFileName  = "terraform.tfvars"
+	TfVarsFileName = "terraform.tfvars"
+
+	// TfPlatformVarsFileName is a template for platform-specific
+	// Terraform variable files.
+	//
+	// https://www.terraform.io/docs/configuration/variables.html#variable-files
+	TfPlatformVarsFileName = "terraform.%s.auto.tfvars"
+
 	tfvarsAssetName = "Terraform Variables"
 )
 
 // TerraformVariables depends on InstallConfig and
 // Ignition to generate the terrafor.tfvars.
 type TerraformVariables struct {
-	File *asset.File
+	files []*asset.File
```

Comment:
cc @staebler

Reply:
This needs to be a public field for consistency between loading from on-disk files and loading from the asset-store state file.

Reply:
> This needs to be a public field...

Fixed with c1c8686 -> e5707ed.